.gitlab-ci.yml

```diff
@@ -233,6 +233,7 @@ gcp-stg:
     - rake rotate_tfstate_key
     - rake rotate_secrets_key
     - rake
+    - rake test_couch_read
```

Comment:
Same here. Yes, it will add 5 minutes to build time, but it will help establish the performance baseline for DB layer independently of other moving parts.